packages/eas-json/src/index.ts

```diff
@@ -1,6 +1,7 @@
 export { AndroidReleaseStatus, AndroidReleaseTrack, SubmitProfile } from './submit/types';
 export { getDefaultProfile as getDefaultSubmitProfile } from './submit/resolver';
 export { EasJson, ProfileType, AppVersionSource } from './types';
+export { Platform } from '@expo/eas-build-job';
```

Comment:
Without exporting that type I can't call `getBuildProfileAsync` unless I call it with as any